### ISSUE TITLE
jtk-endpoint-connected not assigned to endpoints with programmatic connections 

### ISSUE THREAD

**Issue by panayotisk** [jsplumb/jsplumb#1123]
**version**: jsplumb 5.5.5
**description**: When issuing a connect() call the endpoints do not get the class jtk-endpoint-connected. We might need this class to style endpoints via CSS. When connecting using the mouse (after addSourceSelector() and addTargetSelector() calls) the class is assigned to the endpoints.
**code sandbox**: https://codesandbox.io/s/jtk-endpoint-connected-pj9wox

**Comment by panayotisk** [comment]
A (rather dirty) workaround that calls the internal jsplumb function _refreshEndpoint() could be used in the fiddle code. After the connect call one may use:
```
instance._refreshEndpoint(connection.endpoints[0]);
instance._refreshEndpoint(connection.endpoints[1]);
```
where connection is the returned connection of the connect() call.
However since the workaround uses internal functions the issue is still valid.

**Comment by sporritt** [comment]
also applies to the `jtk-endpoint-full` class, which should be assigned if the endpoint has a max connections of 1.

**Comment by sporritt** [comment]
fixed in 5.6.0, released yesterday
